src/cephadm/cephadm

```diff
@@ -1076,6 +1078,16 @@ def infer_fsid(func):
 
     return _infer_fsid
 
+def _get_default_image():
+    if DEFAULT_IMAGE_IS_MASTER:
+        yellow = '\033[93m'
+        end = '\033[0m'
+        s = '{}Warning: This is a development version of cephadm. Please refer to https://docs.ceph.com/docs/{}/cephadm/install{}'.format(
+            yellow, LATEST_STABLE_RELEASE, end
+        )
+        print(s)
```

Comment:
```suggestion
        warn = '''This is a development version of cephadm.

For information regarding the latest stable release:
    https://docs.ceph.com/docs/{}/cephadm/install
'''.format(LATEST_STABLE_RELEASE)
        for line in warn.splitlines():
            logger.warning('{}{}{}'.format(yellow, line, end))
```


* add whitespace to the warning message ?
* use `logger.warning(..)` instead of `print(..)`

![cephadm_warn](https://user-images.githubusercontent.com/1752663/79244096-b565c900-7e33-11ea-8e8e-280e02b13b99.png)

Reply:
that's fancy!

Reply:
applied in https://github.com/ceph/ceph/pull/34693